doc/release/0.17.0-notes.rst

```diff
@@ -37,6 +37,17 @@ are deprecated. These are related to ANOVA, for which ``scipy.stats`` provides
 quite limited functionality and these functions are not very useful standalone.
 See issues #660 and #650 for details.
 
+``scipy.stats.chisqprob`` is deprecated. This is a delegator function and it is
+deprecated in favor of the convention, which is to call the distribution from 
+``scipy.stats.distribution`` or calling ``scipy.special.chdtrc`` directly.
+See issues #2741 and #641 for details.
+
+
+``scipy.stats.betai`` is deprecated. This is delegator function and it is 
+deprecated in favor of the convention, which is to call the distribution from 
+``scipy.stats.distribution`` or calling ``scipy.special.betainc`` directly. 
+See issues #2741 and #642 for details.
```

Comment:
For ``betai`` there is no equivalent method of a distribution instance, so I wouldn't mention that. Also no need to refer to the old issues, I don't think they're that helpful (in release notes we want to point users in the right direction, not explain in detail the background of why we deprecated something).